api/libretimeapi/models.py

```diff
@@ -0,0 +1,663 @@
+# This is an auto-generated Django model module.
+# You'll have to do the following manually to clean this up:
+#   * Rearrange models' order
+#   * Make sure each model has one field with primary_key=True
+#   * Make sure each ForeignKey and OneToOneField has `on_delete` set to the desired behavior
+#   * Remove `managed = False` lines if you wish to allow Django to create, modify, and delete the table
```

Comment:
I agree - Django automatic migrations make things much easier. I don't know how well Propol will play with that though?

Reply:
If done properly we would ensure that everyone applies proper migrations one last time and then have Django take over.

We would still need to keep the actual Propel classes in sync with future changes to the API but doing things like changing field sized on the database can be handled with `manage.py makemigration` instead of manually writing upgrade/downgrade scripts..

I can take a look at this after we merge this PR. Currently a-lot of the db migration happens as a side-effect when the frontend is called. Refactoring this makes for a great chance to get rid of that!

Reply:
Perhaps something we can do with alpha.9->alpha.10 along with the systemd unit renames